.github/workflows/ci.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          submodules: recursive
```

Comment:
I am adding that part because without that it was not loading the added submodule.